### PR TITLE
say something about the limit

### PR DESCRIPTION
**Pull request by andhesmi**

### CODE REVIEW

File: Skills.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This API currently supports versions 4.1 and 4.2. Version is passed in the Accep
 | language            | No | A string determining the language (total 22 languages supported) in which the input text is written. Default value is en. Note that the input parameter passing to language should be the language id. |
 | threshold      | No | A double value between 0 and 1 controlling minimum relevancy scores for skill recognition. Higher values will more tightly restrict the returned skill tags. Default is 0.5. A threshold of 0 means all seed skill phrases recognized by exact string matching will be returned. Note that this parameter is only supported for inputs in English. |
 | auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words, the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en,fr, and de. |
```
Comment (thomasmahoney) [inline Skills.md]: just noticed we're missing a space at the end here ("en,fr, and de")

File: Skills.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This API currently supports versions 4.1 and 4.2. Version is passed in the Accep
...
 | auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words, the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en,fr, and de. |
-| return_relevant_skills | No | A boolean value indicating whether to return relevant skills for each extracted skill. Defaults to false. Relevant skills are identical to extracted skills, with the confidence rating their relevance to the extracted skill. |
+| return_relevant_skills | No | A boolean value indicating whether to return relevant skills for each extracted skill. Defaults to false. Relevant skills are identical to extracted skills, with the confidence rating their relevance to the extracted skill. Up to 10 relevant skills are returned. |
```
Comment (thomasmahoney) [inline Skills.md]: the new sentence is fine, but actually can we change the wording a bit in the preceding sentence? "Relevant skills are identical to extracted skills" is a bit confusing, and "with the confidence rating" reads a little ambiguously (I thought "confidence rating" was meant to be its own thing). I might do something like "Relevant skills contain the same fields as extracted skills, with confidence scores indicating their relevance to the extracted skill."

**Comment by thomasmahoney** [comment]
:+1:

**Comment by andhesmi** [comment]
:+1:

**Comment by thomasmahoney** [comment]
👍

**Pull request merged**